lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1997,6 +2027,112 @@ SPIRVValue *LLVMToSPIRV::transIntrinsicInst(IntrinsicInst *II,
     }
     return Value;
   }
+  case Intrinsic::experimental_constrained_fadd: {
+    auto UnDecBM = BM->addBinaryInst(OpFAdd, transType(II->getType()),
```

Comment:
`UnDecBM` is not really a describing name, could you please choose another one? I suggest to either use some generic one like `BI` for `BinaryInst` or use particular name in each `case` to match with intrinsic (like `FAdd`, `FSub`, etc.)